.changeset/clever-rice-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+'@envelop/sentry': minor
```

Comment:
I didn't noticed, but this PR contains a breaking change, so the version bump should be `major`
```suggestion
'@envelop/sentry': major
```

The breaking change is that now, this plugin wraps the execution phase. So the plugin is now order dependent in the plugin list. Which means it can break existing configuration.

Can you add an advice to put the plugin at the end of the plugin list ? This way, it ensures that the execute function doesn't get overwritten by another plugin.

Reply:
I have changed it to a major update and added a note that it is a breaking change.
[9fb31e0](https://github.com/n1ru4l/envelop/pull/2323/commits/9fb31e0ed742b65876c1999b95e1bffc54df13b4)